client/lib/form/post.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import { forOwn } from 'lodash';
+
+export default function postForm( url, formData ) {
+	const form = document.createElement( 'form' );
+	form.setAttribute( 'method', 'POST' );
+	form.setAttribute( 'action', url );
+
+	forOwn( formData, ( value, name ) => {
+		const field = document.createElement( 'input' );
+		field.setAttribute( 'type', 'hidden' );
```

Comment:
Yeah I don't think it matters much, both fields value are accessible from javascript anyway